lib/models/download.js

```diff
@@ -30,5 +31,18 @@ schema.plugin(scopeChecks);
 schema.plugin(filterByOrg);
 schema.plugin(addCRUDFunctions);
 
+schema.pre('remove', async function removeFile(next) {
+  if (this.upload) {
+    try {
+      const repo = factory(this.upload.repo);
+      await repo.deleteFromPath(this.upload.key);
+    } catch (err) {
+      console.error('models.download remove', err);
```

Comment:
Let's change this to use `logger.error` (e.g. https://github.com/LearningLocker/learninglocker/blob/16bc883bb4af301d74d503d34677ca4c04aa6def/api/src/utils/responses.js#L9)

Reply:
Logger will need importing, see that file as an example